### PR TITLE
full move to osg based image

### PR DESCRIPTION
**Pull request by ivukotic**
These are all the changes needed and there are a lot of them. Some are directory names, variable names, added heartbeat etc.

### CODE REVIEW

File: incubator/cms-xcache/cms-xcache/templates/deployment.yaml [hunk 1]
```diff
@@ -100,9 +100,9 @@ spec:
             value: "{{ .Values.XCacheConfig.Prefetch }}"
           - name: XC_ROOTDIR
             value: "{{ .Values.XCacheConfig.LocalRoot }}"
-          - name: WQ_BLOCKS_PER_LOOP
+          - name: XC_WQ_BLOCKS_PER_LOOP
```
Comment (Mansalu) [inline incubator/cms-xcache/cms-xcache/templates/deployment.yaml:103]: This file is for a different application `cms-xcache`. Did you mean to edit it?

**Review by Mansalu — CHANGES_REQUESTED** [review]

**Comment by ivukotic** [comment]
Hi,

If you are deploying OSG image, than this is the variable name configuration will expect.

Cheers,
Ilija

**Comment by Mansalu** [comment]
Could you bump it's Chart.yaml version also?

https://github.com/ivukotic/slate-catalog/blob/master/incubator/cms-xcache/cms-xcache/Chart.yaml

File: incubator/cms-xcache/cms-xcache/templates/deployment.yaml (hunk 1 above)
Comment (ivukotic) [reply incubator/cms-xcache/cms-xcache/templates/deployment.yaml]: reverted... just to be on a safe side. Please keep in mind that the new OSG image will have these variables with "XC_" prepended.

**Review by Mansalu — APPROVED** [review]
LGTM

**Pull request merged**